app_tests/integration_tests/llm/shortfin/open_llama_3b_llm_server_test.py

```diff
@@ -64,7 +64,7 @@ def test_basic_generation_input_ids(
                 message=f"Generation did not match expected pattern.\nExpected to start with: {expected_prefix}\nActual response: {response}",
             )
 
-    @pytest.mark.parametrize("concurrent_requests", [2, 4, 8])
+    @pytest.mark.parametrize("concurrent_requests", [2, 4])
```

Comment:
We probably also want some soak tests (https://en.wikipedia.org/wiki/Soak_testing), e.g. run for 30 minutes with a variable number of requests coming in throughout that period. These tests are unlikely to catch memory leaks or concurrency issues.

I'd check what vLLM, SGLang, and llama.cpp do too.

Reply:
#1044